dbfit-java/informix/src/integration-test/resources/deinstall_informix.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# Run as root to deinstall Informix and any existing database server.
+# Exit statuses: -
+# 0 - deinstalled successfully.
+# 1 - error occured.
+MODNAME=`basename $0`
+IM="$MODNAME: INFO:"
+EM="$MODNAME: ERROR:"
+WM="$MODNAME: WARNING:"
+
+DBFIT_ROOT=/var/dbfit
+INFORMIXDIR=/informix
+INFORMIX_SCRIPTS=$DBFIT_ROOT/dbfit-java/informix/src/integration-test/resources
+
+if [ ! -x "$INFORMIXDIR/uninstall/uninstall_ids" ]
+then
+	echo "$EM cannot find Informix deinstaller $INFORMIXDIR/uninstall/uninstall_ids" >&2
+	exit 1
+fi
+
+. $INFORMIXDIR/dbfitifserver.ksh
```

Comment:
Is that safe enough? Sounds like `Korn Shell` script being sourced inside bash (we have `#!/usr/bin/env bash` on 1st line). Maybe it's OK, it depends on the script content.